config/locales/letters_en.yml

```diff
@@ -4,52 +4,58 @@ en-GB:
     reject:
       reject_over_savings_html: |
         <p>Dear %{full_name}</p>
+        <p><strong>Reference: %{reference}</strong></p>
         <p>We have received your application for help with fees. Unfortunately you’re not eligible and will have to pay the full fee. This is because you have more than the maximum amount of savings and investments allowed.
         We’ve calculated this as follows:</p>
         <p>Fee to pay:%{fee_amount}</p>
         <p>Maximum amount of savings and investments allowed: %{savings_threshold}</p>
         <p>Your savings and investments total: %{savings_total}</p>
         <p><strong>If you disagree with this decision</strong></p>
         <p>If you disagree with our decision, you can appeal to %{name_delivery_manager}, Delivery Manager at %{court_tribunal_name}. You need to do this by %{date_for_appeal}.</p>
```

Comment:
This letter is not currently displayed, but we'll need to remove the reference to delivery manager and court tribunal too.